### ISSUE TITLE
Would like to have a bit more spacing between recent projects

### ISSUE THREAD

**Issue by Dean-NC** [luminous-software/start-page-plus#30]
### Installed product versions
- Visual Studio: 2019 Community 16.3.2
- This extension: 0.14.3

### Description
I would like to have a bit more **vertical** spacing between recent projects on the "Start" tab.

### Current behavior
Recent projects on the "Start" tab are slightly too vertically close to each other.

### Expected behavior
Recent projects would have a bit more vertical spacing.

**Comment by yannduran** [comment]
Yeah, I wavered over this. No matter what I do, I'm not going to please everyone unfortunately. I still have UI tweaking to do nearer to v1.0, so I haven't settled on exactly how I want it to look yet.

I do plan on adding a number of configuration settings, so it sounds like this would be a good one to add to them.

**Comment by Dean-NC** [comment]
Thanks. I think it's low priority, but for me having a little more space makes project names stand out/pop, especially when there are a few projects together that start with the same words, like "MyCustomer.Project1.Part1", "MyCustomer.Project1.Part2", etc. Also, I'm guessing the list will scroll, so no worry about a little extra space causing any issue getting to other projects.

**Comment by yannduran** [comment]
Thanks for the feedback. And yes, the list scrolls.

**Comment by yannduran** [comment]
@Dean-NC 

Now that **v1.0** has been released, do you still feel the same way?
